test/FunctionalTests/Microsoft.OData.Core.Tests/IntegrationTests/Evaluation/AutoComputePayloadMetadataInJsonIntegrationTests.cs

```diff
@@ -441,71 +439,71 @@ static AutoComputePayloadMetadataInJsonIntegrationTests()
                 ContainsTarget = true
             });
 
-            var container = new EdmEntityContainer("Namespace", "Container");
-            EntitySet = container.AddEntitySet("EntitySet", EntityType);
+            var serviceProvider = new EdmEntityContainer("Namespace", "Container");

```

Comment:
```suggestion
            var entityContainer = new EdmEntityContainer("Namespace", "Container");
```
Why did we change `container` of type `EdmEntityContainer` into `serviceProvider`?